Source/App/EbAppConfig.c

```diff
@@ -95,6 +95,7 @@
 #define ASM_TYPE_TOKEN				    "-asm" // no Eval
 #define THREAD_MGMNT                    "-lp"
 #define TARGET_SOCKET                   "-ss"
+#define SWITCHTHREADSTOREALTIME_TOKEN   "-switchThreadsToRtPriority"
```

Comment:
Could we use a shorter name for the api? internally it is fine to use a more readable name.

Reply:
How about "-real-time" or "-real-time-thread"?

Reply:
much better, or "-rt"?

Reply:
set to "-rt"